binding/react/src/useRhino.ts

```diff
@@ -127,16 +117,28 @@ export function useRhino(
       };
     }
 
+    const {
+      accessKey,
+      context,
+      requireEndpoint,
+      start: startWebVp = true,
+    } = rhinoHookArgs!;
+    if (accessKey === null || accessKey === '') {
+      return (): void => {
+        /* NOOP */
```

Comment:
is this `safe`?

Reply:
I don't see why not, the standard return is a function these ones are just empty.